README.md

```diff
@@ -18,7 +18,9 @@ and open source.
 
 For announcements about new releases and breaking changes, follow the
 [flutter-announce@googlegroups.com](https://groups.google.com/forum/#!forum/flutter-announce)
-mailing list.
+mailing list or see the
+[breaking changes](https://flutter.dev/docs/release/breaking-changes) page of
+Flutter's official website.
```

Comment:
I'd probably remove the 'of Flutter's official website' clause. End the sentence at 'page'.

Reply:
Thanks, done.